tools/dataMigration/migrateActivityLog.ts

```diff
@@ -0,0 +1,26 @@
+import { Cycle } from '../../src/meta/assessment'
+import { Assessment } from '../../src/meta/assessment/assessment'
+import { BaseProtocol } from '../../src/server/db'
+import { DBNames } from './_DBNames'
+
+export const migrateActivityLog = async (props: { assessment: Assessment }, client: BaseProtocol): Promise<void> => {
+  const { assessment } = props
+  const cycle = assessment.cycles.find((cycle: Cycle) => cycle.name === '2020')
+
+  const schemaName = DBNames.getAssessmentSchema(assessment.props.name)
+  const query = `
+      insert into ${schemaName}.activity_log (
+          time, message, country_iso, section, target, user_id, cycle_uuid
+      )
+      select
+          time,
+          message,
```

Comment:
Are you 100% sure the message , and all other columns match ? I suspect they don't 
At least the message should be updates

Reply:
I forgot about this, I must check; maybe @rfontanarosa  can help me match to correct ones as he is the author of the new ones

Reply:
ok, let's talk about it tomorrow after the daily